pom.xml

```diff
@@ -94,6 +101,17 @@
             <ratchetFrom>origin/lts-2025</ratchetFrom>
           </configuration>
         </plugin>
+        <plugin>
+          <groupId>com.github.eirslett</groupId>
+          <artifactId>frontend-maven-plugin</artifactId>
+          <version>1.15.1</version>
+          <configuration>
+            <nodeVersion>${node.version}</nodeVersion>
+            <nodeDownloadRoot>${node.download.root}</nodeDownloadRoot>
```

Comment:
FYI we've set up https://nexus.platform.dev.nuxeo.com/repository/nodejs-upstream/